cdk/domino_cdk/config/eks.py

```diff
@@ -29,12 +29,15 @@ class NodegroupBase:
         key_name: some-key-pair - Pre-existing AWS key pair to configure for instances in the nodegorup
         min_size: 1 - Minimum node count for nodegroup. Can't be 0 on managed nodegroups.
         max_size: 10 - Maximum limit for node count in node gorup
-        machine_image: AMI and/or user_data script for nodes in nodegroup. When overriding AMI, you MUST
-                       setup your custom AMI or your user_data script _MANUALLY_ to join the cluster.
-                       Additionally, options that use user_data (ie ssm_agent) are no longer valid.
-                       machine_image:
-                         ami_id: ami-123abc
-                         user_data: ...
+        ami_id: ami-123abc - AMI to use for nodegroup, empty/null will use the default EKS AMI, which
+                             is the default. When specifying an AMI, you MUST specify a custom user_data
+                             script to join the node to the cluster, and this script must do any sort of
+                             node setup that is desired. Options that are implemented via user_data (ie
+                             labels, taints, ssm_agent) are not supported with custom AMIs, and must be
```

Comment:
This reads as "user_data options are not supported with custom AMI"

-> "Labels, taints and ssm_agent config options are not supported with custom AMIs, and must be implemented manually via user_data"

Reply:
I was trying to describe the reason the options are not supported. I updated the language to try and convey it without any room for ambiguity.